src/http/server.ts

```diff
@@ -84,25 +94,29 @@ export class Server {
    * @description
    *     A property to hold middleware.
    *
-   * @property any middleware
+   * @property middleware
    */
   protected middleware: {
     before_request?: Array<
-      ((request: any) => Promise<void>) | ((request: any) => void)
+      | ((request: Drash.Http.Request) => Promise<void>)
+      | ((request: Drash.Http.Request) => void)
     >;
     after_request?: Array<
-      | ((request: any, response: Drash.Http.Response) => Promise<void>)
-      | ((request: any, response: Drash.Http.Response) => void)
+      | ((
+        request: Drash.Http.Request,
+        response: Drash.Http.Response,
+      ) => Promise<void>)
+      | ((request: Drash.Http.Request, response: Drash.Http.Response) => void)
```

Comment:
Maybe we could create an interface and then we could type it in the docs?

Reply:
i think we should do this in another issue. it was hard as fuck to type